### ISSUE TITLE
macOS Big Sur Little Snitch 5 Network Module Conflicting With Adguard's Module, Mail SMTP Timing Out

### ISSUE THREAD

**Issue by XxUnkn0wnxX** [AdguardTeam/AdguardForMac#776]
The issue is that Adguards network extension is clashing with little snitches network Extention, even turning off little snitch's monitor does not fix it. if I remove little snitch from network settings it works just fine, but then I need to reboot to enable little snitch again so it re-installs the profile into the network settings.

if I disable adguard mail SMTP works fine, disabling rules or any other settings within adguard does not mitigate the issue, only if I use proxy mode instead of the network module.

also, unticking mail in applications does not fix the issue.
<img width="662" alt="Screen Shot 2020-11-14 at 4 42 34 pm" src="https://user-images.githubusercontent.com/2242880/99140727-7a14cc80-2698-11eb-9311-5619374eb260.png">

#  Steps to reproduce
install little snitch 5.0.2 & have it running
make sure it looks like this in your network settings:
<img width="174" alt="Screen Shot 2020-11-14 at 4 32 35 pm" src="https://user-images.githubusercontent.com/2242880/99140650-beec3380-2697-11eb-9c32-a3b80087f146.png">
then open mail & make sure you have icloud setup
go Window --> Connection Doctor

### Expected behavior
icloud SMTP to connect without issues.

### Actual behavior
icloud SMTP keeps timing out:
<img width="842" alt="Screen Shot 2020-11-14 at 4 32 40 pm" src="https://user-images.githubusercontent.com/2242880/99140675-ea6f1e00-2697-11eb-95ca-4ccdede4dba3.png">

### Customer ID 
I didn't get an auto-reply, but I included the issue ID in the message I sent with the report.

### Your environment
* Environment name and version: macOS Big Sur, Apple Mail App
* Any specific potentially conflicting software installed: Little Snitch 5.02

**Comment by Eugene-Savenko** [comment]
HeldDesk ID: 3017724

**Comment by Chinaski1** [comment]
Hello there!

To troubleshoot this issue, we need to get the app logs.

Here's what we need you to do:

1. Menu -> Gear -> Advanced -> Logging level -> Debug;
2. Reproduce the issue (and remember the exact time when it happened);
3. Menu -> Advanced -> Export Logs and System Info...;
4. Send the archive to **`apple@adguard.com`** and mention this issue number in the subject.

**Comment by XxUnkn0wnxX** [comment]
email sent...

**Comment by XxUnkn0wnxX** [comment]
after sending the logs, I took a look at the smtp logs for iCloud & it uses the domain p47-smtp.mail.me.com with the port 587.

I added that in the advanced settings

<img width="912" alt="Screen Shot 2020-11-18 at 2 08 23 am" src="https://user-images.githubusercontent.com/2242880/99407445-372a5180-2943-11eb-820b-b2512688a547.png">

& the issue has resolved while using the network extension.

also when I removed the new changes I added the issue does not occur anymore so I'm not sure what happened. maybe after upgrading to big sur without doing a clean install of adgurad the settings did not persist until re-applying them?
